prov/usnic/src/usdf_ep_rdm.c

```diff
@@ -662,12 +662,26 @@ usdf_ep_rdm_bind_cq(struct usdf_ep *ep, struct usdf_cq *cq, uint64_t flags)
 static int
 usdf_ep_rdm_bind(struct fid *fid, struct fid *bfid, uint64_t flags)
 {
+	int ret;
 	struct usdf_ep *ep;
 	struct usdf_cq *cq;
 	struct usdf_av *av;
 
 	USDF_TRACE_SYS(EP_CTRL, "\n");
 
+	/* Backward compatibility case for Open MPI. We haven't been validating the flags until now.
+	 * Older version of Open MPI gives FI_RECV as AV bind flag (bug). */
```

Comment:
Let's remove the backwards compat phrase here from the RDM code, since Open MPI doesn't currently use RDM with usNIC.